MPSGameLift/Documentation/GameLiftLocal.md

```diff
@@ -0,0 +1,53 @@
+# MultiplayerSample Project with Amazon GameLift Local
+
+This README covers testing and running MultiplayerSample with Amazon GameLift Local.
+
+## Local testing with [GameLiftLocal](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-testing-local.html)
```

Comment:
The markdown preview for this file isn't working.

Do you if it is correctly formatted markdown?
The VSCode markdown previewer can be used to check if the markdown works there.